data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/S3ScanBucketOption.java

```diff
@@ -25,23 +22,20 @@ public class S3ScanBucketOption {
     @JsonProperty("name")
     private String name;
 
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
-    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonDeserialize(using = CustomLocalDateTimeDeserializer.class)
     @JsonProperty("start_time")
     private LocalDateTime startTime;
 
-    @JsonSerialize(using = LocalDateTimeSerializer.class)
     @JsonDeserialize(using = LocalDateTimeDeserializer.class)
     @JsonProperty("end_time")
     private LocalDateTime endTime;
 
-    @JsonSerialize(using = DurationSerializer.class)
-    @JsonDeserialize(using = DurationDeserializer.class)
+    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
```

Comment:
Why is this using a `LocalDateTimeDeserializer` instead of `DurationDeserializer`?

Reply:
Good catch. I will make the change.